modules/augment/include/opencv2/augment/averageblur.hpp

```diff
@@ -50,6 +38,8 @@ private :
     Size maxKernelSize;
     Size kernelSize;
     bool sameXY;
+    std::vector<int> borderTypes;
+    int borderType;
```

Comment:
Please avoid duplication

Reply:
This is not a duplicatio.. borderType is the current random instance of borderTypes

Reply:
Oh, sorry. That's OK.